operators/multiclusterobservability/controllers/multiclusterobservability/grafana.go

```diff
@@ -55,9 +55,10 @@ type GrafanaDatasource struct {
 }
 
 type JsonData struct {
-	TLSAuth               bool   `yaml:"tlsAuth,omitempty"`
-	TLSAuthCA             bool   `yaml:"tlsAuthWithCACert,omitempty"`
-	QueryTimeout          string `yaml:"queryTimeout,omitempty"`
+	TLSAuth   bool `yaml:"tlsAuth,omitempty"`
```

Comment:
Nit, but would be nice to keep the formatting here is consistent.

Reply:
Ack, it is formatted correctly via gofmt however the comment on this particular field throws off the indentation. Since this is a bug fix I'd just keep it minimal here as is, but I intend on doing a follow up which reviews and explains all the fields in this file which will address this!